cmd/juju/controller/listcontrollers_test.go

```diff
@@ -28,10 +29,8 @@ var _ = gc.Suite(&ListControllersSuite{})
 
 func (s *ListControllersSuite) TestListControllersEmptyStore(c *gc.C) {
 	s.store = jujuclienttesting.NewMemStore()
-	context, err := s.runListControllers(c)
-	c.Assert(err, jc.ErrorIsNil)
-	c.Check(testing.Stdout(context), gc.Equals, "")
-	c.Check(testing.Stderr(context), gc.Equals, modelcmd.ErrNoControllersDefined.Error())
+	_, err := s.runListControllers(c)
+	c.Check(err.Error(), gc.Equals, modelcmd.ErrNoControllersDefined.Error())
```

Comment:
Since the command no longer logs at the info level to standard error (but instead returns its error up to a higher level) we cannot check standard error here. We instead check that the appropriate error was returned.